### PR TITLE
Add Javax dep for Java 11

### PR DESCRIPTION
**Pull request by carl-mastrangelo**

### CODE REVIEW

**Comment by carl-mastrangelo** [comment]
Hi, this is needed to upgrade Netflix/zuul to Java 11.   The jar being added in was factored out of Java8 and into a separate package, thus the need to include the dep.

**Comment by qiangdavidliu** [comment]
@carl-mastrangelo do you mind re pushing your two PRs? They are both failing on the builder JDK. The change to open should fix that. Thanks.

**Comment by carl-mastrangelo** [comment]
@qiangdavidliu done PTAL.

Also as a side note, governator-test didn't compile for me.  The gradle file is missing several of the dependencies.

**Comment by qiangdavidliu** [comment]
@asibross

**Comment by carl-mastrangelo** [comment]
cc: @artgon

**Comment by carl-mastrangelo** [comment]
Friendly ping, anything more to do here?

**Comment by carl-mastrangelo** [comment]
FYI I updated a few more things to work with Java 11.   These should be pretty minor.  I confirmed this builds with both Java 11 and Java 8.

PTAL.   Also, I don't know what procedure is for merging, I assume you merge after you approve.

File: governator-jersey/build.gradle [hunk 1]
```diff
@@ -14,6 +14,10 @@ dependencies {
 
     compileOnly 'javax.servlet:javax.servlet-api:3.0.1'
 
+    implementation "javax.xml.bind:jaxb-api:2.3.1"
```
Comment (asibross) [inline governator-jersey/build.gradle]: I understand that jaxb was removed in Java 11, but where in governator code is it being used, or which of it's dependencies uses it?

File: governator-jersey/build.gradle [hunk 2]
```diff
@@ -14,6 +14,10 @@ dependencies {
...
+    runtime "com.sun.xml.bind:jaxb-core:2.3.0.1"
+    runtime "com.sun.xml.bind:jaxb-impl:2.3.2"
```
Comment (asibross) [inline governator-jersey/build.gradle]: If Jaxb is not being used but it's annotation are loaded when a class is loaded, it would suffice to just add the jaxb-api.
Also, an alternative to the split package jaxb implementation is `jaxb-runtime` from glassfish.
https://www.jesperdj.com/2018/09/30/jaxb-on-java-9-10-11-and-beyond/

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,6 @@
+#Thu Oct 10 13:23:52 PDT 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
```
Comment (asibross) [inline gradle/wrapper/gradle-wrapper.properties]: Why is this change needed? *-all instead of *-bin.

File: governator-core/build.gradle [hunk 4]
```diff
@@ -2,6 +2,7 @@ apply plugin: 'java'
 
 dependencies {
     compile     project(':governator-api')
+    compile     "javax.annotation:javax.annotation-api:1.3.2"
```
Comment (brharrington) [inline governator-core/build.gradle]: This project is no longer maintained as it is part of the EE stuff that was transitioned to Eclipse:

https://github.com/javaee/javax.annotation/blob/master/README.md

The version of this package released by eclipse is:

https://mvnrepository.com/artifact/jakarta.annotation/jakarta.annotation-api/1.3.5

File: governator-jersey/build.gradle (hunk 1 above)
Comment (carl-mastrangelo) [reply governator-jersey/build.gradle]: Unit tests.  If you run the tests they pass, but complain about the missing classes.  I'm guessing the test is broken, but appears passing.

File: gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (carl-mastrangelo) [reply gradle/wrapper/gradle-wrapper.properties]: This is what ./gradlew build did.  Not sure why.   Reverting it is probably a fruitless endeavor.

**Comment by asibross** [comment]
Changes are covered by #400
